foundation/shared-services/core/aks.tf

```diff
@@ -14,6 +14,7 @@ data "azurerm_private_dns_zone" "aks_private_dns_id" {
 }
 
 data "azurerm_log_analytics_workspace" "log_analytics_workspace" {
+  provider            = azurerm.management
```

Comment:
`provider` Doesn't seem to be a valid attribute https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/log_analytics_workspace

Reply:
This isn't a resource attribute for the log analytics workspace, but instead a meta-argument. https://www.terraform.io/language/providers/configuration#selecting-alternate-provider-configurations

I think this is valid (and tested)